src/Agent.Listener/MessageListener.cs

```diff
@@ -103,6 +103,7 @@ public override void Initialize(IHostContext hostContext)
                     if (ex is TaskAgentNotFoundException)
                     {
                         Trace.Error("The agent no longer exists on the server. Stopping the agent.");
+                        _term.WriteError(StringUtil.Loc("MissingAgent"));
```

Comment:
Will print on the console for TaskAgentSessionConflictException the same message as in the trace: "The session for this agent already exists.". Should I try to make custom messages  for fatal exceptions: TaskAgentPoolNotFoundException, AccessDeniedException and TaskAgentNotFoundException or just print the message contained within (always English)?